app/models/invitation.rb

```diff
@@ -0,0 +1,4 @@
+class Invitation < ActiveRecord::Base
+  validates :email, presence: { message: "can't be blank" }
+  validates :email, uniqueness: { message: "has already be taken" }
```

Comment:
"has already **been** taken". And grammar apart, there's Russian almost everywhere, why use English here? :)